pages/authed-with-fission.tsx

```diff
@@ -0,0 +1,66 @@
+import styles from '@pages/app.module.scss';
+
+import * as React from 'react';
+import * as U from '@common/utilities';
+
+import { useFissionAuth } from '@common/useFissionAuth';
+
+import Page from '@components/Page';
+import SingleColumnLayout from '@components/SingleColumnLayout';
+import Button from '@components/Button';
+
+import { H1, H2, H3, H4, P } from '@components/Typography';
+
+export async function getServerSideProps(context) {
+  const viewer = await U.getViewerFromHeader(context.req.headers);
+
+  if (viewer) {
+    return {
+      redirect: {
+        permanent: false,
+        destination: '/home',
+      },
+    };
+  }
+
+  return {
+    props: { viewer },
+  };
+}
+
+function AuthedWithFissionPage(props: any) {
+  const [state, setState] = React.useState({
+    loading: false,
+    fs: null,
+  });
+  const { signIn, username } = useFissionAuth();
+
+  return (
+    <Page title="Estuary: Authenticated with Fission" description="Authenticated with Fission. Continue to Estuary" url="https://estuary.tech/authed-with-fission">
+      <SingleColumnLayout style={{ maxWidth: 488 }}>
+        <H2>Authenticated with Fission</H2>
+        <P style={{ marginTop: 16 }}>{username ? `Welcome back! You are signed into Fission as ${username}.` : 'One moment, we are loading your Fission account.'}</P>
+
+        <div className={styles.actions}>
+          <Button
+            style={username ? { width: '100%' } : { width: '100%', backgroundColor: '#aaaaaa' }}
+            loading={state.loading ? state.loading : undefined}
+            onClick={async () => {
+              setState({ ...state, loading: true });
+              const response = await signIn();
+              if (response && response.error) {
+                alert(response.error);
+                setState({ ...state, loading: false });
+              }
+            }}
+          >
+            Continue to Estuary
```

Comment:
Where is this supposed to go?

Reply:
That should redirect to the user's home page after `signIn` is called in `useFissionAuth` (unless there is an error).

https://github.com/application-research/estuary-www/blob/2285c5b79800184ead8292a8ff88a6e6161ea6e9/common/useFissionAuth.ts#L112

There's a bit of indirection to that. Maybe it would be better to find a way to handle that in the page?